index.html

```diff
@@ -134,6 +137,329 @@ <h1 class="banner-header banner-content">
+
+        <p class="body-text-2">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <p class="head-text-1">LET'S CONNECT</p>
+
+        <div class="banner-socials">
+          <ul class="banner-socials">
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/twitter.svg"
+                  alt="Twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/linkedin-in.svg"
+                  alt="Linkedin"
+              /></a>
+            </li>
+
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/medium.svg"
+                  alt="Linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/github.svg"
+                  alt="Github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/discord.svg"
+                  alt="Github"
+              /></a>
+            </li>
+          </ul>
+        </div>
+
+        <div>
+          <button type="button" class="work-card-button button-text-1">See Project</button>
+        </div>
+      </div>
+
+      <!--About collapsible-->
+      <div class="about-collapsible">
+        <ul>
+          <li>
+            <div class="about-collapsible-button">
+              <button
+                type="button"
+                class="collapsible about-collapsible-button head-text-2"
+              >
+                Languages
+                <object
+                  class="icon-rotate"
+                  data="assets/icons/chevron.svg"
+                ></object>
+              </button>
+            </div>
+            <div class="about-collapsible-content">
+              <div class="about-collapsible-tools">
+                <img src="assets/images/tool-js.png" alt="javaScript" />
+                <p class="sub-text-1">JavaScript</p>
+              </div>
+
+              <div class="about-collapsible-tools">
+                <img src="assets/images/tool-html.png" alt="javaScript" />
+                <p class="sub-text-1">HTML</p>
+              </div>
+
+              <div class="about-collapsible-tools">
+                <img src="assets/images/tool-css.png" alt="javaScript" />
+                <p class="sub-text-1">CSS</p>
+              </div>
+            </div>
```

Comment:
According to the evaluation guidelines, this section should be wrapped inside `<ul>` and `<li>` tags as well.

<img width="747" alt="Screen Shot 2022-05-26 at 14 46 29" src="https://user-images.githubusercontent.com/82426682/170565618-cc5b50fc-6eac-49b9-a4f5-bbcb5d6fb44e.png">

Kindly fix this.